content/en/docs/collector/deployment/gateway/index.md

```diff
@@ -198,6 +199,42 @@ The load-balancing exporter emits metrics including
 that you can use for health and performance monitoring of the OTLP endpoint
 collector.
 
+## Typical Gateway deployment architecture
+
+A typical Gateway architecture involves running both Collector in Gateway mode
+and Agent mode.
+
+The following diagram shows a typical deployment architecture for a gateway
+deployment. Here
+
+- We use Agent Collectors (running on each host, similar to Kubernetes
+  daemonsets) to collect telemetry from services running on the host and host
+  telemetry, such as host metrics and scrap logs.
+- We use Gateway Collectors to process OTLP data, such as filtering, sampling,
+  and exporting to backends etc.
+
+![gateway](otel-gateway-arch.svg)
+
+There are a few limitations in running the OpenTelemetry Collector in gateway
+mode.
+
+- There are few receivers in OpenTelemetry Collector that need to be unique per
+  host instance. Running multiple instances of these receivers will result in
+  duplicate data. It is recommended not to use them in the Gateway Collector but
+  only for the collector running in Agent mode. Examples include, but are not
+  limited to:
+
+  - [`hostmetricsreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver),
+    may result in duplicate host metrics.
+  - [`filelogreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver),
+    may result in duplicate logs.
+
+- Using a
+  [`resourcedetection`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor)
+  processor in gateway mode is not recommended. This processor is used to detect
+  the resources of the host where the Collector is running. Use
+  `resourcedetection` processor in the pipeline of Agent collector deployment.
```

Comment:
I would like to rephrase this section here, because it is key for a user to understand why they would need a combination of agents and gateways. This is the reason why this effort is needed, that needs to be made clearer. A suggestion:

```
This combined deployment pattern is necessary, when you use components in your Collector that either need to be unique per host or that consume information that is only available on the same host as the application is running:

- Receivers like the [`hostmetricsreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver) or [`filelogreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) need to be unique per host instance. Running multiple instances of these receivers will result in duplicated data.

- Processors like the [`resourcedetectionprocessor`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor) are used to add information about the host the collector and the application are running on. Running them within a Collector on a remote machine will result in incorrect data.
```